src/ios/PurchasesPlugin+Purchasing.swift

```diff
@@ -177,6 +177,31 @@ import PurchasesHybridCommon
 #endif
     }
 
+    @objc(showInAppMessages:)
+    func showInAppMessages(command: CDVInvokedUrlCommand) {
+        let intMessageTypes = command.argument(at: 0, withDefault: nil) as? [Int]
+#if os(iOS) || targetEnvironment(macCatalyst) || VISION_OS
+        if #available(iOS 16.0, *) {
+            if let intMessageTypes {
+                let messageTypes = intMessageTypes.map({ intNumber in
+                    NSNumber(integerLiteral: intNumber)
+                })
+                CommonFunctionality.showStoreMessages(forRawValues: Set(messageTypes)) { [weak self] in
+                    self?.sendOKFor(command: command)
+                }
+            } else {
+                CommonFunctionality.showStoreMessages { [weak self] in
+                    self?.sendOKFor(command: command)
+                }
+            }
+        } else {
+            self.sendOKFor(command: command)
+        }
+#else
+        self.sendOKFor(command: command)
+#endif
```

Comment:
should we log anything in the `else` cases?

Reply:
Added in https://github.com/RevenueCat/cordova-plugin-purchases/pull/430/commits/79ba3d76944c34d9c19bcb2272d834ae96d18039